File: android-java/app/src/main/java/io/agora/convoai/example/startup/KeyCenter.java
package io.agora.convoai.example.startup;

/**
 * KeyCenter
 * Load values from BuildConfig, which are populated from env.properties at build time
 */
public class KeyCenter {
    public static final String AGORA_APP_ID = BuildConfig.AGORA_APP_ID;
    public static final String AGORA_APP_CERTIFICATE = BuildConfig.AGORA_APP_CERTIFICATE;
    public static final String REST_KEY = BuildConfig.REST_KEY;
    public static final String REST_SECRET = BuildConfig.REST_SECRET;
    public static final String PIPELINE_ID = BuildConfig.PIPELINE_ID;
}

File: reactnative/src/utils/KeyCenter.ts
import { KeyCenterConfig } from './KeyCenterConfig';

export class KeyCenter {
  static get AGORA_APP_ID(): string {
    return KeyCenterConfig.AGORA_APP_ID;
  }

  static get AGORA_APP_CERTIFICATE(): string {
    return KeyCenterConfig.AGORA_APP_CERTIFICATE;
  }

  static get REST_KEY(): string {
    return KeyCenterConfig.REST_KEY;
  }

  static get REST_SECRET(): string {
    return KeyCenterConfig.REST_SECRET;
  }

  static get PIPELINE_ID(): string {
    return KeyCenterConfig.PIPELINE_ID;
  }

  static readonly USER_ID = 1001086;
  static readonly AGENT_RTC_UID = 1009527;
}
